src/Databases/DatabaseReplicatedSettings.h

```diff
@@ -7,13 +7,24 @@ namespace DB
 
 class ASTStorage;
 
-#define LIST_OF_DATABASE_REPLICATED_SETTINGS(M) \
-    M(Float, max_broken_tables_ratio, 0.5, "Do not recover replica automatically if the ratio of staled tables to all tables is greater", 0) \
+#define CONTEMPORARY_DATABASE_REPLICATED_SETTINGS(M) \
+    M(Float,  max_broken_tables_ratio, 0.5, "Do not recover replica automatically if the ratio of staled tables to all tables is greater", 0) \
```

Comment:
double space

Reply:
This is intended to make the code look nice 
<img width="356" alt="image" src="https://user-images.githubusercontent.com/25705399/159757993-2c35f074-1581-44b2-8afe-46483320acf4.png">

Reply:
We don't have such indentation in Settings.h or MergeTreeSettings.h, but ok